ml-proto/test/memory.wasm

```diff
@@ -26,6 +27,17 @@
   (module (memory 100 1000 (segment 0 "a") (segment 2 "b") (segment 1 "c")))
   "data segment not disjoint and ordered")
 
+;; Test alignment annotation rules
+(assertinvalid (module (func (loadu.2.i8 (const.i32 0)))) "alignment too big")
```

Comment:
I was not aware that anyone had discussed this issue; could you provide a link?  This seems like the sort of decision that should be explicitly documented since it's nonobvious whether "power of 2" is just waving its hands at the boundary conditions (as we often do) or really means "no, seriously, every power of 2!" (up to what limit?).  I'm happy to remove this check here for now since ultimately it's a minor detail and people have actually discussed it.